components/Form.svelte

```diff
@@ -5,14 +5,29 @@
     TextArea,
     TextInput,
   } from "carbon-components-svelte";
-  import { gjScheme, clearCurrentlyEditing } from "../stores.js";
+  import {
+    gjScheme,
+    clearCurrentlyEditing,
+    currentlyEditing,
+  } from "../stores.js";
 
   export let id;
   export let name;
   export let intervention_type;
   export let description;
   export let length_meters;
 
+  let bottomOfForm;
+
+  currentlyEditing.subscribe((openedId) => {
+    if (id == openedId) {
+      // Wait 1ms before doing this, because it appears the accordion doesn't
+      // expand instantly. This is flaky when clicking the accordion instead of
+      // the map.
+      setTimeout(() => bottomOfForm?.scrollIntoView({ behavior: "smooth" }), 1);
```

Comment:
The `?` is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining. There could be some crazy race condition where the form vanishes by the time 1ms has passed, so don't blow up if it becomes null